<ui version="4.0" >
 <class>Kleo::NewCertificateUi::EnterDetailsPage</class>
 <widget class="QWizardPage" name="Kleo::NewCertificateUi::EnterDetailsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>204</height>
   </rect>
  </property>
  <property name="title" >
   <string comment="@title" >Enter Details</string>
  </property>
  <property name="subTitle" >
   <string>Please enter your personal details below. If you want more control over the certificate parameters, click on the Advanced Settings button.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QFormLayout" name="formLayout" />
   </item>
   <item>
    <widget class="QLineEdit" name="resultLE" >
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="addEmailToDnCB" >
     <property name="text" >
      <string>Add email address to DN (only needed for broken CAs)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="advancedPB" >
       <property name="text" >
        <string>Advanced Settings...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>advancedPB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::EnterDetailsPage</receiver>
   <slot>slotAdvancedSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>359</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel" >
     <x>459</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAdvancedSettingsClicked()</slot>
 </slots>
</ui>
